test/test/compare-responses.js

```diff
@@ -0,0 +1,98 @@
+const chai = require("chai");
+const expect = chai.expect;
+const fs = require('fs')
+const path = require('path')
+const sanitizeFileName = require("sanitize-filename");
+
+const constants = require('./constants');
+const SAVED_RESPONSES_PATH =  constants.SAVED_RESPONSES_PATH;
+
+/**
+ * Remove variable values such as build time, version, etc. from response.
+ *
+ * This enables us to compare responses from tests.
+ *
+ * @param response response to sanitize
+ * @return response with variable values replaces with fixed ones
+ */
+function sanitizeResponse(response) {
```

Comment:
I have a very similar function at indices.js:clearKeys(). Do you think is worth merging these 2 functions into one?

Reply:
It seems like the appropriate place to put it these functions is actually here

Reply:
Good idea, done!